src/Bundle/CoverallsBundle/Entity/Exception/RequirementsNotSatisfiedException.php

```diff
@@ -72,6 +72,11 @@ public function getHelpMessage()
   - APPVEYOR
   - APPVEYOR_BUILD_NUMBER
 
+For Githib Actions users:
+  - GITHUB_REF
+  - GITHUB_SHA
+  - GITHUB_ACTIONS
```

Comment:
Only `COVERALLS_REPO_TOKEN` is needed. The rest is set by GA.

Reply:
See https://github.com/php-coveralls/php-coveralls/blob/2ecb7c15392c83a2c7e889382290fa3dd9d574a2/src/Bundle/CoverallsBundle/Entity/Exception/RequirementsNotSatisfiedException.php#L56, this variable was setted by Travis, but she has in error message

Reply:
But I'm using it without those variables explicitly set so I'm not sure we should require users to do so.

Reply:
https://github.com/webonyx/graphql-php/blob/611f5b04704fe6901737c973d052caaf275bc00f/.github/workflows/ci-build.yml#L158-L159 this seems to be enough

Reply:
I understand you, but it's linked with https://github.com/php-coveralls/php-coveralls/pull/275#pullrequestreview-338427212, if maintainer answer on my question about required variable for GA

Reply:
@simPod i add new function `requireGithubActions` that can work without using `COVERALLS_RUN_LOCALLY`. See https://github.com/Smolevich/laravel-mongodb/commit/adb57c22b5358673945c9dc452ab7e94ee9e6a33/checks?check_suite_id=405300949

Reply:
Users don't set variables like `GITHUB_SHA` in ci file, it part of environment for ci build. We can descriprion to README for example ci in GA